DashAI/back/core/config.py

```diff
@@ -71,6 +74,7 @@ class Settings(BaseSettings):
     FRONT_BUILD_PATH: str = os.path.join(dashai_path, "front/build")
     USER_DATASET_PATH: str = os.path.join(dashai_path, "back/user_datasets")
     USER_RUN_PATH: str = os.path.join(dashai_path, "back/user_runs")
+    USER_EXPLAINER_PATH: str = os.path.join(dashai_path, "back/user_explainers")
```

Comment:
```suggestion
```

Not used in the new version with dependency injection. 

To migrate to the new version: 

1. add the new path to the pydantic config DefaultSettings, located in DashAI/back/config.py:

`GLOBAL_EXPLAINERS_PATH: str = "global_explainers". `


2. In DashAI/back/app.py:_generate_config_dict, create a new APP path with:

`settings["GLOBAL_EXPLAINER_PATH"] = local_path / settings["GLOBAL_EXPLAINER_PATH"]`

2. Create the path with:

`_create_path_if_not_exists(container.config.provided()["GLOBAL_EXPLAINERS_PATH"])`

The above will define the default explainers folder as  `~/.DashAI/global_explainers`